packages/admin/cms-admin/src/cronJobs/CronJobsGrid.tsx

```diff
@@ -66,11 +66,17 @@ export function CronJobsGrid() {
     const intl = useIntl();
     const client = useApolloClient();
     const stackSwitchApi = useStackSwitchApi();
+    const [isOpen, setIsOpen] = useState(false);
+    const [cronJob, setCronJob] = useState("");
```

Comment:
You could remove the isOpen state by using the cron job to start as toggle for the dialog:

```ts
const [cronJobToStart, setCronJobToStart] = useState<string>();
const dialogOpen = Boolean(cronJobToStart);
```